api/tests/test_cloud_record_summary.py

```diff
@@ -18,8 +15,134 @@ class CloudRecordSummaryTest(TestCase):
     """Tests GET requests to the Cloud Sumamry Record endpoint."""
 
     def setUp(self):
-        """Disable logging.INFO from appearing in test output."""
-        logging.disable(logging.INFO)
+        """Prevent logging from appearing in test output."""
+        logging.disable(logging.CRITICAL)
+
+    def test_cloud_record_summary_get_IAM_fail(self):
+        """
+        Test what happens if we fail to contact the IAM.
+
+        i.e, _token_to_id returns None
+        """
+        CloudRecordSummaryView._token_to_id = Mock(return_value=None)
```

Comment:
it gets used in by CloudRecordSummaryView class which handles the get requests. So we don't actually see it being used in the test case, but it simulates the IAM being un-contact-able.

I've added a comment to that effect.